src/test/java/eu/bbmri_eric/negotiator/unit/service/PostServiceTest.java

```diff
@@ -215,6 +216,7 @@ void after() throws Exception {
   }
 
   @Test
+  @Disabled
```

Comment:
Why have you disabled this?

Reply:
Because the Service tests that mock all other components are too brittle (you add a method call to a repository or a service and all the tests that test that method need to be changed)

Reply:
But in this way you're not testing the method at all

Reply:
the method is covered by the integration tests no?

Reply:
but we have a whole test class for the Post Controller which really just calls the Service methods

Reply:
They are two different tests. The controller test tests (or should test) that when the service raises ForbiddenRequestException the controller returns 403. The service's test checks the events that cause the PostService to raise that exception. I think that keeping them is worthwhile.

Reply:
well currently it also tests the application logic (e.g. if a post is created). Should we split the tests so that Service Integration tests cover business logic while the Controller tests cover only serialization and HTTP errors?

Reply:
because I would rather rewrite the whole test class to use SpringBootTest rather than mocks

Reply:
You can add the SpringBootTest for integration tests but I don't see why disabling the unit tests that cover specific points.  BTW, why these two?